qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -36,7 +40,13 @@
 class IBMQBackend(BaseBackend):
     """Backend class interfacing with an IBMQ backend."""
 
-    def __init__(self, configuration, provider, credentials, api):
+    def __init__(
+            self,
+            configuration: BackendConfiguration,
+            provider,
+            credentials: Credentials,
+            api: IBMQConnector
```

Comment:
Can you double-check this declaration? As part of the transition between the old and the new api, a good rule of thumb is that, most of the time, the objects that expected an `IBMQConnector` also accept the equivalent in the new API (client or similar). In general, we cannot rely solely on the existing docstrings, as in this case it seems it was not updated.

Reply:
Yeah, thanks for that catch Diego! I'll double check to make sure it is of the right type in that case.